Gemfile

```
@@ -11,6 +11,7 @@ gem "bootsnap", ">= 1.4.4", require: false
 
 # Asset related
 gem 'sass-rails', '>= 6'
+gem "sprockets", "3.7.2"
```

I don't know why I had to downgrade but I keep getting:

```
SassC::SyntaxError: Error: Undefined variable
```

I have it downgrade based on:

* https://stackoverflow.com/questions/62233567/rails-6-on-heroku-sasscsyntaxerror-error-undefined-variable-body-bg#62238810 and
* https://github.com/sass/sassc-rails/issues/122#issuecomment-607007731

Any suggestion and help is very welcome :)

---

```suggestion
gem "sprockets", "< 4"
```

I've had to do this for a few projects too

---

> I've had to do this for a few projects too

Thanks :D. Done :tada: